service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/RaiseQueryController.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.hmcts.reform.fpl.controllers;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.enums.CaseRole;
+import uk.gov.hmcts.reform.fpl.enums.UserRole;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.service.UserService;
+
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.fpl.utils.QueryManagementUtils.getCurrentCollectionByUserService;
+import static uk.gov.hmcts.reform.fpl.utils.QueryManagementUtils.getLatestQueryIDForCollection;
+import static uk.gov.hmcts.reform.fpl.utils.QueryManagementUtils.getRoleToCollectionMapping;
+import static uk.gov.hmcts.reform.fpl.utils.QueryManagementUtils.logAllQueryCollections;
+
+@Slf4j
+@RestController
+@RequestMapping("/callback/raise-query")
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class RaiseQueryController extends CallbackController {
+
+    private final UserService userService;
+    private static final Map<CaseRole, String> COLLECTION_MAPPING = getRoleToCollectionMapping();
```

Comment:
shall we move this constant to the util class? so if somewhere else need to access the mapping, they don't have to build the map again. 
(but yeah... static variable are initialised before run time, so wouldn't harm if we leave it here)